src/ncp/thread_host.hpp

```diff
@@ -85,8 +85,9 @@ class NetworkProperties
 class ThreadHost : virtual public NetworkProperties
 {
 public:
-    using AsyncResultReceiver = std::function<void(otError, const std::string &)>;
-    using DeviceRoleHandler   = std::function<void(otError, otDeviceRole)>;
+    using AsyncResultReceiver  = std::function<void(otError, const std::string &)>;
+    using ChannelMasksReceiver = std::function<void(int, int)>; // Supported Channel Mask, Preferred Channel Mask.
```

Comment:
The [definition](https://cs.android.com/android/platform/superproject/main/+/main:external/ot-br-posix/src/android/aidl/com/android/server/thread/openthread/IChannelMasksReceiver.aidl?q=IChannelMasksReceiver) in android is `int` so I used `int` here. I'm not sure why is that. But let's keep consistent to avoid casting. Thoughts?

Reply:
Currently, these interfaces are only used by Android, but these interfaces can be expanded to other platforms in the future as needed. We can convert the `uint32_t` to `int` when passing the parameters to Android interface.

Reply:
SGTM. Updated!